LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 gitric
```

Comment:
This should be `Copyright (c) 2020 SIL International`

Reply:
done